robottelo/manifests.py

```diff
@@ -232,7 +232,7 @@ def upload_manifest_locked(org_id, manifest=None, interface=INTERFACE_API, timeo
         # And as we are in locked state, other functions/tests can try to upload the manifest in
         # other processes and we do not want to be interrupted by the default configuration
         # ssh_client timeout.
-        timeout = 1500
+        timeout = 1500000
```

Comment:
isn't this timeout so huge only for testing purpose ?

Reply:
It's not huge.  @JacobCallahan updated time to be in milliseconds so this has to be updated to a factor of 1000 for it to actually work.  It was something he may have missed in his original unstable PR that he did a week back.

Reply:
The timeout should actually be set per interface. seconds is fine for API, but CLI is in milliseconds or "smart" timeout format